getpaid/backends/transferuj/__init__.py

```diff
@@ -68,19 +70,26 @@ def online(ip, id, tr_id, tr_date, tr_crc, tr_amount, tr_paid, tr_desc, tr_statu
         payment.external_id = tr_id
         payment.description = tr_email
 
-        if tr_status == 'TRUE':
-            # Due to Transferuj documentation, we need to check if amount is correct
-            payment.amount_paid = Decimal(tr_paid)
-            payment.paid_on = datetime.datetime.utcnow().replace(tzinfo=utc)
-            if payment.amount <= Decimal(tr_paid):
-                # Amount is correct or it is overpaid
-                payment.change_status('paid')
-            else:
-                payment.change_status('partially_paid')
-        elif payment.status != 'paid':
+        signals.payment_rejection_available.send(sender=None, payment=payment)
```

Comment:
I don’t need this PR anymore, so from my perspective it can be
closed. On the other, if something is broken here, it is probably
Transferuj’s notifications handling itself, not the payment rejection
feature, so merging this PR theoretically shouldn’t make things worse.